modules/codelite/tests/test_codelite_additional_rules.lua

```diff
@@ -60,7 +60,7 @@
 			value = 0
 		}
 
-		buildmessage 'Rule-ing %{file.name}'
+		buildmessage "Rule'ing %{file.name}"
```

Comment:
Why did `-` change to `'`? Seems like an unintended/unnecessary change?

Reply:
It was to show/test single quote escaping.
I can revert that part if you prefer.

Reply:
That should be done in a dedicated test for escaping, as mentioned in my other comment, as this looks like a typo and will likely end up being changed back eventually.

Reply:
Roll backed.